internal/autoupdate/pages/static/list.css

```diff
@@ -16,49 +15,80 @@ a {
 }
 
 body {
-  font-family:  ui-monospace,SFMono-Regular,SF Mono,Consolas,Liberation Mono,Menlo,monospace;
+  font-family: ui-monospace, SFMono-Regular, "SF Mono", Consolas, "Liberation Mono", Menlo, monospace;
   line-height: 1.2;
 }
 
-table[class="queue_table"] {
+table#queue-table {
   border: 1px solid #D9D7CE;
-  width: 80ch;
+  width: 100%;
+  table-layout: fixed;
   font-size: 1.1em;
   background-color: #eeeeee;
   padding: 5px;
   text-align: left;
 }
 
-table[class="queue_table"] th {
+table#queue-table th {
   border: 10px solid black;
   background-color: black;
   color: white;
 }
 
+table#queue-table tr:nth-child(odd) {
+}
 
-table tr[class="tr_active_prs"] {
+table#queue-table tr.active-prs {
   background-color: #eeeeee;
 }
 
-table tr[class="tr_suspended_prs"] {
-  background-color: #cccccc ;
+table#queue-table tr.suspended-prs {
+  background-color: #cccccc;
 }
 
-table td[id="td_status_updating"] {
-  color: #0eb200;
+table#queue-table td {
+  padding: 5px;
+  overflow: hidden;
+}
+
+table#queue-table th.rank, table#queue-table td.rank {
+  width: 50px;
+}
+
+table#queue-table th.pr, table#queue-table td.pr {
+  width: auto;
+  word-break: break-all;
```

Comment:
this is to break words regardless, to avoid that cases such as:

```
[AUT-580] qa/automation: Disable TestCertificateUnderinsurancePaymentverifyUnderInsurancePaymentLinkIsDisabled test
```

stretch into eternity breaking the layout.